scripts/release/package_desktop_artifact.py

```
@@ -61,100 +61,85 @@ def resolved_artifact_target() -> tuple[str, str]:
-        "\n".join(
-            [
-                f"platform={identity['platform']}",
-                f"arch={identity['arch']}",
-                f"target_triple={os.environ.get('BANDSCOPE_TARGET_TRIPLE', 'native')}",
-                f"binary={binary_path.name}",
-                f"archive={archive_name}",
-                f"checksum={checksum_path.name}",
-            ]
+    installers = find_installer_packages(repo_root)
+    if not installers:
+        raise FileNotFoundError("Could not find any built installers (DMG/EXE) in target/release/bundle/")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)